exercises/isogram/.meta/generator/isogram_case.rb

```diff
@@ -6,9 +6,19 @@ def workload
     indent_lines(
       [
         "string = #{input.inspect}",
-        "#{assert} Isogram.is_isogram?(string)"
+        "#{assert} Isogram.isogram?(string), #{failure_message}"
       ], 4
     )
   end
 
+  private
+
+  def failure_message
+    %Q("Expected '#{expected}', '#{input}' #{type}")
+  end
+
+  def type
+    "#{expected ? 'is' : 'is not'} an isogram"
```

Comment:
```ruby
  def failure_message
    %Q("Expected '#{expected}', '#{input}' #{reason}")
  end

  def reason
    "#{is_or_not} an isogram"
  end

  def is_or_not
    expected ? 'is' : 'is not'
  end
```
Since we aren't really getting a type, the method name is not quite right.